ext/native/gfx_es2/draw_buffer.cpp

```diff
@@ -402,35 +425,38 @@ void DrawBuffer::MeasureTextCount(int font, const char *text, int count, float *
 			// Ignore lone ampersands
 			continue;
 		}
-		const AtlasChar *c = atlasfont.getChar(cval);
+		const AtlasChar *c = atlasfont->getChar(cval);
 		if (c) {
 			wacc += c->wx * fontscalex;
 		}
 	}
 	if (w) *w = std::max(wacc, maxX);
-	if (h) *h = atlasfont.height * fontscaley * lines;
+	if (h) *h = atlasfont->height * fontscaley * lines;
 }
 
-void DrawBuffer::MeasureTextRect(int font, const char *text, int count, const Bounds &bounds, float *w, float *h, int align) {
-	if (!text || (uint32_t)font >= (uint32_t)atlas->num_fonts) {
+void DrawBuffer::MeasureTextRect(FontID font_id, const char *text, int count, const Bounds &bounds, float *w, float *h, int align) {
+	if (!text || font_id.isInvalid()) {
 		*w = 0;
 		*h = 0;
 		return;
 	}
 
 	std::string toMeasure = std::string(text, count);
 	if (align & FLAG_WRAP_TEXT) {
-		AtlasWordWrapper wrapper(*atlas->fonts[font], fontscalex, toMeasure.c_str(), bounds.w);
+		const AtlasFont *font = atlas->getFont(font_id);
+		if (!font)
+			return;
```

Comment:
Hm, should still set w/h to 0?

-[Unknown]

Reply:
Yup, will fix.